src/coreModules/api/apiClient/index.js

```diff
@@ -3,53 +3,95 @@ const createApiMethod = require('./createApiMethod')
 
 module.exports = function createApiClient(options) {
   const apiConfig = createApiConfig(options)
+
+  const formPost = createApiMethod(apiConfig, {
+    mapHeaders: userInputHeaders => {
+      return {
+        ...userInputHeaders,
+        'Content-Type': 'application/x-www-form-urlencoded',
+      }
+    },
+    method: 'POST',
+  })
+
+  const httpDelete = createApiMethod(apiConfig, {
+    method: 'delete',
+  })
+
+  const httpGet = createApiMethod(apiConfig, {
+    method: 'GET',
+  })
+
+  const httpPatch = createApiMethod(apiConfig, {
+    mapHeaders: userInputHeaders => {
+      return {
+        ...userInputHeaders,
+        'Content-Type': 'application/json',
+      }
+    },
+    method: 'PATCH',
+  })
+
+  const httpPost = createApiMethod(apiConfig, {
+    mapHeaders: userInputHeaders => {
+      return {
+        ...userInputHeaders,
+        'Content-Type': 'application/json',
+      }
+    },
+    method: 'POST',
+  })
+
+  const httpPut = createApiMethod(apiConfig, {
+    mapHeaders: userInputHeaders => {
+      return {
+        ...userInputHeaders,
+        'Content-Type': 'application/json',
+      }
+    },
+    method: 'POST',
```

Comment:
should be `PUT`

...why don't we use lower case so we don't have to remember the special case with `delete`? also the switch expects `methodName` to be in lower case.